doc/api/fs.md

```diff
@@ -898,6 +898,21 @@ try {
 }
 ```
 
+```cjs
+const { access, constants } = require('node:fs/promises');
+
+async function checkAccess() {
+  try {
+    await access('/etc/passwd', constants.R_OK | constants.W_OK);
+    console.log('can access');
```

Comment:
This doesn't seem right to me, actually. Specifically, I think the `console.log()` can trigger despite the access failing.